plugins/azure/resoto_plugin_azure/resource/base.py

```diff
@@ -609,6 +646,10 @@ def edges_of(
                 if isinstance(from_node, from_type) and isinstance(to_node, to_type) and key.edge_type == edge_type
             ]
 
+    @property
+    def config(self) -> AzureConfig:
+        return self._config
```

Comment:
We can't do that, because decorator `@property` will return that method itself. We should set property `config` as `self._config`, to return it correctly.